lib/web/jquery/jquery.storageapi.min.js

```diff
@@ -1,2 +1,2 @@
ethod:function(e,t){var r=[this._type],t=Array.prototype.slice.call(t),i=t[0];return this._ns&&r.push(this._ns),"string"==typeof i&&-1!==i.indexOf(".")&&(t.shift(),[].unshift.apply(t,i.split("."))),[].push.apply(r,t),e.apply(this,r)},get:function(){return this._callMethod(t,arguments)},set:function(){var t=arguments.length,i=arguments,n=i[0];if(1>t||!e.isPlainObject(n)&&2>t)throw Error("Minimum 2 arguments must be given or first parameter must be an object");if(e.isPlainObject(n)&&this._ns){for(var o in n)r(this._type,this._ns,o,n[o]);return n}var s=this._callMethod(r,i);return this._ns?s[n.split(".")[0]]:s},remove:function(){if(arguments.length<1)throw Error("Minimum 1 argument must be given");return this._callMethod(i,arguments)},removeAll:function(e){return this._ns?(r(this._type,this._ns,{}),!0):n(this._type,e)},isEmpty:function(){return this._callMethod(o,arguments)},isSet:function(){if(arguments.length<1)throw Error("Minimum 1 argument must be given");return this._callMethod(s,arguments)},keys:function(){return this._callMethod(a,arguments)}};if(e.cookie){window.name||(window.name=Math.floor(1e8*Math.random()));var l={_cookie:!0,_prefix:"",_expires:null,_path:null,_domain:null,setItem:function(t,r){e.cookie(this._prefix+t,r,{expires:this._expires,path:this._path,domain:this._domain})},getItem:function(t){return e.cookie(this._prefix+t)},removeItem:function(t){return e.removeCookie(this._prefix+t)},clear:function(){for(var t in e.cookie())""!=t&&(!this._prefix&&-1===t.indexOf(c)&&-1===t.indexOf(m)||this._prefix&&0===t.indexOf(this._prefix))&&e.removeCookie(t)},setExpires:function(e){return this._expires=e,this},setPath:function(e){return this._path=e,this},setDomain:function(e){return this._domain=e,this},setConf:function(e){return e.path&&(this._path=e.path),e.domain&&(this._domain=e.domain),e.expires&&(this._expires=e.expires),this},setDefaultConf:function(){this._path=this._domain=this._expires=null}};g||(window.localCookieStorage=e.extend({},l,{_prefix:c,_expires:3650}),window.sessionCookieStorage=e.extend({},l,{_prefix:m+window.name+"_"})),window.cookieStorage=e.extend({},l),e.cookieStorage=e.extend({},h,{_type:"cookieStorage",setExpires:function(e){return window.cookieStorage.setExpires(e),this},setPath:function(e){return window.cookieStorage.setPath(e),this},setDomain:function(e){return window.cookieStorage.setDomain(e),this},setConf:function(e){return window.cookieStorage.setConf(e),this},setDefaultConf:function(){return window.cookieStorage.setDefaultConf(),this}})}e.initNamespaceStorage=function(e){return u(e)},g?(e.localStorage=e.extend({},h,{_type:"localStorage"}),e.sessionStorage=e.extend({},h,{_type:"sessionStorage"})):(e.localStorage=e.extend({},h,{_type:"localCookieStorage"}),e.sessionStorage=e.extend({},h,{_type:"sessionCookieStorage"})),e.namespaceStorages={},e.removeAllStorages=function(t){e.localStorage.removeAll(t),e.sessionStorage.removeAll(t),e.cookieStorage&&e.cookieStorage.removeAll(t),t||(e.namespaceStorages={})}});
```

Comment:
Please revert changes in this file, it’s not related to this PR

Reply:
It is though, this library is extremely out of date and had a fix issued in version 1.7.5 (Magento has 1.7.3) that fixes an issue that prevents unit tests from being executable and has a rare chance to throw in error in general.

It was fixed in 2015, I placed it in the file as a means of correcting the issue: https://github.com/julien-maurel/jQuery-Storage-API/commit/abef0586cf92e29297a283ee6299bac741ae125e#diff-50c51a23ea1ca042039829fb4ddeeb7c

This library had it's final release in May of 2016 (1.9.1) I think it's kind of ridiculous that this library is on a 2014 version, I'm also fine upgrading this library to 1.7.5 or even 1.9.1 (not sure how much divergence there is between 1.7.5 & 1.9.1)

Please advise how you want me to proceed.

Reply:
@complexthings 
> issue that prevents unit tests from being executable and has a rare chance to throw in error in general.

I think fixing or even updating some library just to fix the test is not good thing. Could you explain what the issue you had in the test? Maybe we'll find some workaround for that?

Reply:
I removed it in a commit, you'll be able to see below the results of Magento's Unit Tests running.  But essentially here's what happens:

![image](https://user-images.githubusercontent.com/1484134/88667583-9243ea00-d0af-11ea-96f1-944e9618029d.png)

Reply:
I just tried to investigate.
In original code we have
https://github.com/dariusz-goc/jQuery-Storage-API/blob/abef0586cf92e29297a283ee6299bac741ae125e/jquery.storageapi.js#L428 that runs `_createNamespace` method.

This method sets https://github.com/dariusz-goc/jQuery-Storage-API/blob/abef0586cf92e29297a283ee6299bac741ae125e/jquery.storageapi.js#L278

something like this:
```
    var ns={
      localStorage:$.extend({},$.localStorage,{_ns:name}),
      sessionStorage:$.extend({},$.sessionStorage,{_ns:name})
    };
    ...
    $.namespaceStorages[name]=ns;
```

Looks like we need to put some mock `$.namespaceStorages[name]` and issue should be solved. What do you thing?

Reply:
In general this plugin is buggy for instance it causes issues like https://github.com/magento/magento2/issues/27355 and https://github.com/magento/magento2/issues/17195, would be good to get rid of it at some point, but not in scope of this task.

Reply:
@complexthings please check my comments above, hope it will fix the issue you had